### PR TITLE
TC04_Contact Email contains correct href value which can help to quickly open Outlook Compose Message dialog

### PR DESCRIPTION
**Pull request by thinhhp96**

### CODE REVIEW

**Review by fey031 — APPROVED** [review]
LGTM

File: src/main/java/page_objects/ContactPage.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package page_objects;
+
+import helpers.BrowserHelper;
+import org.openqa.selenium.By;
+
+public class ContactPage extends BasePage {
+    private final By lnkEmail = By.linkText("thanh.viet.le@logigear.com");
```
Comment (tanmle) [inline src/main/java/page_objects/ContactPage.java]: nooooo, you want to check for `mailto:thanh.viet.le@logigear.com`, but you capture an element with that value inside, find another way.

File: src/test/java/tests/TC04_CheckContactEmailAddressHref.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    @Test(description = "Contact Email contains correct href value which can help to quickly open Outlook Compose Message dialog")
+    public void TC04() {
+        contactPage.gotoContactPage();
+
+        String actualEmailHref = contactPage.getEmailHref();
+        String expectedEmailHref = "mailto:thanh.viet.le@logigear.com";
+
+        Assert.assertEquals(actualEmailHref, expectedEmailHref, "Email address's href is not match!");
+    }
+
```
Comment (tanmle) [inline src/test/java/tests/TC04_CheckContactEmailAddressHref.java]: redundant

**Review by tanmle — CHANGES_REQUESTED** [review]

File: src/main/java/page_objects/ContactPage.java (hunk 1 above)
Comment (thinhhp96) [reply src/main/java/page_objects/ContactPage.java]: I've captured with new CSS Selector. Please check and review it. Thank you!

File: src/test/java/tests/TC04_CheckContactEmailAddressHref.java (hunk 2 above)
Comment (thinhhp96) [reply src/test/java/tests/TC04_CheckContactEmailAddressHref.java]: I've deleted this line.

**Review by tanmle — APPROVED** [review]

**Pull request merged**